File: 02 templates/template.cpp
// ========================================
// Competitive Programming Template
// Author: [Your Name]
// ========================================

#include <bits/stdc++.h>
using namespace std;

// ============ TYPE DEFINITIONS ============
typedef long long ll;
typedef long double ld;
typedef pair<int, int> pii;
typedef pair<ll, ll> pll;
typedef vector<int> vi;
typedef vector<ll> vll;
typedef vector<pii> vpii;
typedef vector<pll> vpll;

// ============ MACROS ============
#define ALL(x) (x).begin(), (x).end()
#define RALL(x) (x).rbegin(), (x).rend()
#define PB push_back
#define MP make_pair
#define F first
#define S second
#define SZ(x) (int)(x).size()
#define REP(i, n) for (int i = 0; i < (n); i++)
#define FOR(i, a, b) for (int i = (a); i < (b); i++)
#define FORE(i, a, b) for (int i = (a); i <= (b); i++)
#define ROF(i, a, b) for (int i = (a); i >= (b); i--)

// ============ CONSTANTS ============
const int INF = 1e9 + 7;
const ll LINF = 1e18;
const int MOD = 1e9 + 7;
const ld PI = acos(-1.0);
const ld EPS = 1e-9;

// ============ FAST I/O ============
void setup() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);
#ifdef LOCAL
    freopen("input.txt", "r", stdin);
    freopen("output.txt", "w", stdout);
#endif
}

// ============ DEBUG ============
#ifdef LOCAL
#define debug(x) cerr << #x << " = " << (x) << endl
#define debug2(x, y) cerr << #x << " = " << (x) << ", " << #y << " = " << (y) << endl
#else
#define debug(x)
#define debug2(x, y)
#endif

// ============ UTILITY FUNCTIONS ============
template <typename T>
void print_vector(const vector<T> &v) {
    for (const auto &x : v)
        cout << x << " ";
    cout << "\n";
}

// ============ MATH UTILITIES ============
ll mod_pow(ll base, ll exp, ll mod) {
    ll result = 1;
    while (exp > 0) {
        if (exp % 2 == 1)
            result = (result * base) % mod;
        base = (base * base) % mod;
        exp /= 2;
    }
    return result;
}

ll mod_inv(ll a, ll mod) {
    return mod_pow(a, mod - 2, mod);
}

ll gcd(ll a, ll b) {
    return b == 0 ? a : gcd(b, a % b);
}

ll lcm(ll a, ll b) {
    return a / gcd(a, b) * b;
}

// ============ MAIN SOLUTION ============
void solve() {
    // Your solution here
}

int main() {
    setup();

    int t = 1;
    cin >> t; // Comment this line for single test case

    while (t--) {
        solve();
    }

    return 0;
}

File: 02 templates/template.py
#!/usr/bin/env python3
# ========================================
# Competitive Programming Template (Python)
# Author: [Your Name]
# ========================================

import sys
import math
from collections import defaultdict, deque, Counter
from heapq import heappush, heappop, heapify
from bisect import bisect_left, bisect_right
from itertools import combinations, permutations, accumulate
from functools import lru_cache, reduce

# ============ FAST I/O ============
def read_int():
    return int(sys.stdin.readline())

def read_ints():
    return map(int, sys.stdin.readline().split())

def read_int_list():
    return list(map(int, sys.stdin.readline().split()))

def read_str():
    return sys.stdin.readline().strip()

def read_str_list():
    return sys.stdin.readline().split()

# ============ CONSTANTS ============
INF = float('inf')
MOD = 10**9 + 7
EPS = 1e-9

# ============ MATH UTILITIES ============
def mod_pow(base, exp, mod):
    """Fast modular exponentiation"""
    result = 1
    base %= mod
    while exp > 0:
        if exp & 1:
            result = (result * base) % mod
        base = (base * base) % mod
        exp >>= 1
    return result

def mod_inv(a, mod):
    """Modular inverse using Fermat's Little Theorem"""
    return mod_pow(a, mod - 2, mod)

def gcd(a, b):
    """Greatest Common Divisor"""
    while b:
        a, b = b, a % b
    return a

def lcm(a, b):
    """Least Common Multiple"""
    return a * b // gcd(a, b)

def is_prime(n):
    """Primality test"""
    if n < 2:
        return False
    if n == 2:
        return True
    if n % 2 == 0:
        return False
    for i in range(3, int(math.sqrt(n)) + 1, 2):
        if n % i == 0:
            return False
    return True

def sieve(n):
    """Sieve of Eratosthenes"""
    is_prime = [True] * (n + 1)
    is_prime[0] = is_prime[1] = False
    for i in range(2, int(math.sqrt(n)) + 1):
        if is_prime[i]:
            for j in range(i * i, n + 1, i):
                is_prime[j] = False
    return [i for i in range(n + 1) if is_prime[i]]

# ============ MAIN SOLUTION ============
def solve():
    """Solve one test case"""
    # Your solution here
    pass

def main():
    """Main function"""
    t = read_int()  # Number of test cases
    # t = 1  # Uncomment for single test case
    
    for _ in range(t):
        solve()

if __name__ == "__main__":
    main()
